spec/Cabin/InteriorLights.vspec

```diff
@@ -54,8 +54,13 @@
   type: sensor
   description: Is a shared light across a specific row on
 
-- Spotlight.IsPassengerOn:
+- Spotlight.IsLeftOn:
```

Comment:
Spotlight is Dome Light within the cabin?

Reply:
The spec is named InteriorLights, so from my perspective it must mean this.  But at the same time I do think that it would each fully qualified node should have a description that can be understood on its own.  So it is a good idea to be quite detailed in the description I mean.